ddocs/medic-client/views/contacts_by_freetext/map.js

```diff
@@ -1,5 +1,16 @@
 function(doc) {
-  var skip = [ '_id', '_rev', 'type', 'refid' ];
+  var skip = [ '_id', '_rev', 'type', 'refid', 'geolocation' ];
+
+  var usedKeys = [];
+  var emitMaybe = function(key, value) {
+    if (usedKeys.indexOf(key) === -1 && // Not already used
```

Comment:
Maybe just pass the `word` string in to `emitMaybe` instead of the array - they're all just single word keys anyway and it'll avoid having to say `[0]` all the time.